pkg/strvals/parser.go

```diff
@@ -54,9 +54,9 @@ func Parse(s string) (map[string]interface{}, error) {
 //
 // If the strval string has a key that exists in dest, it overwrites the
 // dest version.
-func ParseInto(s string, dest map[string]interface{}) error {
+func ParseInto(s string, dest map[string]interface{}, stringBoolean bool) error {
```

Comment:
I think it may be clearer and won't break the current API if you add a new method called `ParseIntoString`. The other private methods can still use a `bool`, but for someone consuming this package, they won't need to change how they are calling the methods. There might even be a better idea from one of the other maintainers as well. `ParseIntoString` seems pretty simple though